### ISSUE TITLE
DONATION FOR THE LORD.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#11405]
|<img alt="serena2o03@...'s avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| serena2o03@... reported|
|-|-|
|Trac ID|trac#304|
|Type|enhancement|
|Created|2003-10-01 03:29:15Z|

<details><summary>Searchable metadata</summary>

```
trac-id__304 304
type__enhancement enhancement
reporter__serena2o03____ serena2o03@...
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1064978955000000 1064978955000000
changetime__1468232316609926 1468232316609926
version__ 
owner__ 
cc__serena2o03@...
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="serena2o03@...'s avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|serena2o03@... commented|
|-|-|

```
#!html
<pre>
From: Mrs Serena Jones

PLEASE ENDEAVOUR TO USED IT FOR THE CHILDREN OF GOD.
I am the above named person from Kuwait. I am married to Dr.Harry Jones who worked with Kuwait embassy in Ivory Coast for nine yearsbefore he died in the year 2000.We were married for eleven years without a child. He died after a brief illness that lasted for only four days. Before his death we were both born again Christians.Since his death I decided not to re-marry or get a child outside my  matrimonial home which the Bible is against.When my
late husband was alive he deposited the sum of$8.6Million (Eight Millionsix hundred  thousand U.S. Dollars) with one finance/security company in Amsterderm Holland. Presently, this money is still with the Security Company. Recently, my Doctor told me that I would not last for the next
three months due to cancer problem. Though what disturbs me most is my  stroke sickness. Having known my condition I decided to donate this fund to church or better still a christian individual that will utilize this money the way I am going to instruct here in. I want a church that will
use this fund to fund churches, orphanages and widows propagating the word of God and to ensure  that the house of God is maintained. The Bible made us to understand that Blessed is the hand that giveth.
I took this decision because I don't have any child that will  inherit this money and my husband relatives are not Christians and I don't want my husband's hard earned money to be misused by unbelievers. I don'twant a situation where this money will be used in an ungodly manner.
Hence the reason for taking this bold decision. I am not afraid of death hence I know where I am going. I know that I am going to be in the bosom of the Lord. Exodus 14 VS 14 says that  the lord will fight my case and I shall hold my peace. I don't need any telephone communication in this
regard because of my health because of the presence of my husband's relatives around me always. I don't want them to know about this development. 
With God all things are possible.  As soon as I receiveyour reply I shall give you the contact of the Finance/Security Company in Amsterderm Holland. I will also issue you a letter of authority that will prove you as the original- beneficiary of this fund. I want you and the church to always pray for me because the lord is my shephard. My 
happiness is that I lived a life of a worthy Christian. Whoever that wants to serve the Lord must serve him in spirit and truth. Please always be prayerful all through your life. Any delay  in your reply will give me room in sourcing for a chuch or christian individual for this
same purpose. Please assure me that you will act accordingly as I stated herein.  Hoping to hearing from you. N.B-PLEASE I WILL ADVICE YOU TO GIVE THE LAWYER IN CHARGE A CALL IN HOLLAND IMMEDIATELY, HE DOES EVERYTHING ON MY BEHALF AND HE'S VERY UNDERSTANDING AND I BELIEVE HE WILL LEAD YOU TO YOUR SUCCESS IN JESUS 
NAME:      James Vincent
Email:     jamesvin01@...

Remain blessed in the name of the Lord.
Yours in Christ,
Mrs Serena Jones  
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
Roor suck ugh ugh ugh we're getting spam bugs :(
marking resolved
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="moshez's avatar" src="https://avatars.githubusercontent.com/u/422703?s=50" width="50" height="50">](https://github.com/moshez)|@moshez commented|
|-|-|

```
#!html
<pre>
roor more spam
resolving
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub commented|
|-|-|

In changeset f9381660d3964e77e0b517ff1f227a389147669a
```
#!CommitTicketReference repository="" revision="f9381660d3964e77e0b517ff1f227a389147669a"
Merge pull request #304 from twisted/8578-rodrigc-tox-win32

Author: rodrigc
Reviewer: Lukasa
Fixes: #8578

Fix invoking tox on Windows to run tests
```
